app/code/core/Mage/Adminhtml/Block/Sales/Order/Shipment/Packaging.php

```diff
@@ -171,8 +170,7 @@ public function getDeliveryConfirmationTypeByCode($code)
         if ($carrier) {
             $params = new Varien_Object(array('country_recipient' => $countryId));
             $confirmationTypes = $carrier->getDeliveryConfirmationTypes($params);
-            $confirmationType = !empty($confirmationTypes[$code]) ? $confirmationTypes[$code] : '';
-            return $confirmationType;
+            return !empty($confirmationTypes[$code]) ? $confirmationTypes[$code] : '';
```

Comment:
can I suggest to write: `return empty($confirmationTypes[$code]) ? '' : $confirmationTypes[$code];`

Reply:
Will fixed in separate PR.